content/docs/notebooks/custom-notebook.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Create a Custom Jupyter Image"
+description = "Creating a custom Docker image for your Jupyter notebook"
+weight = 30
++++
+
+This guide tells you how to configure a custom Docker image for your Jupyter
+notebook server in Kubeflow.
+
+Your custom image must meet the requirements of the Kubeflow notebook
+controller which manages the life cycle of notebooks. The Kubeflow UI expects
+Jupyter to start after launching the Docker image with `docker run`. You must
+therefore set the default command of your Docker image to launch Jupyter.
+
+Follow these steps to configure your Docker image:
+
+* Set the working directory:
+
+    ```
+    --notebook-dir=/home/jovyan
+    ```
+
+    The `/home/jovyan`  directory is backed by a 
+    [Kubernetes persistent volume (PV)](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
+
+* Allow Jupyter to listen on all IP addresses:
+
+    ```
+    --ip=0.0.0.0
+    ```
+
+* Allow the user to run the notebook as root:
+
+    ```
+    --allow-root
+    ```
+
+* Set the port:
+
+    ```
+    --port=8888
+    ```
+
+* Disable authentication, as Kubeflow takes care of authentication. Use the 
```

Comment:
I read this sentence as, you need to disable authentication in JupyterHub because Kubeflow takes care of authentication for your. Which I interpret as, users accessing JupyterHub will be authenticated by Kubeflow. But, I don't see anything in the docs or in the UI about managing authentication.

This leads me to think that the Kubeflow UI is providing a proxy to the JupyterHub endpoint, and cannot authenticate with JupyterHub. Does that sound correct?

I'm not sure if there is a change that is necessary here, but I think that using the term "authentication" may lead users to think about how their users will authenticate with Kubeflow when accessing JupyterHub.

Reply:
@zabbasi Please would you clarify and suggest how I should adjust the wording.

(@joeliedtke  Just a note that we're not using JupyterHub any more. I think I've changed all the docs that mentioned JupyterHub, but if you've spotted something please let me know.)

Reply:
Done. PTAL.